src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValues.java

```diff
@@ -109,8 +112,20 @@ Object returnValueFor(Class<?> type) {
             return new LinkedHashMap<Object, Object>();
         } else if ("java.util.Optional".equals(type.getName())) {
             return JavaEightUtil.emptyOptional();
+        } else if ("java.util.OptionalDouble".equals(type.getName())) {
```

Comment:
Your change is really good, it improves Mockito behavior for Java8!

Regardless of the final decision whether to merge it or not, I find it disturbing that it wasn't discussed as a breaking change in this PR. @bric3, @TimvdLippe thoughts?